requirements.txt

```diff
@@ -481,9 +481,9 @@ django-structlog==9.0.0 \
     --hash=sha256:0ada1ac0fa4e90a499fc9ce22fc164830304e7219cab12d3b5ef9c2f2967ff5b \
     --hash=sha256:8c32097b3f09b2727b746d6cdf4f50565c62fd040b3a0c98cb731de0b380b854
     # via -r requirements.in
-django-tables2==2.7.1 \
-    --hash=sha256:1ac6426b05dc4b537ba3a952d2920c51653302ca7dcbd1d320cf6e421eeb2dbf \
-    --hash=sha256:d96f70884889197a28d91d1b006f997de3d8c3bcf867828fb2ed54bd86d38807
+django-tables2==2.7.4 \
```

Comment:
@jefftriplett @saadmk11 I can't find any reference to this package other than in the requirements files. Should we remove it?

Reply:
django-tables2 is used here: https://github.com/djangopackages/djangopackages/blob/main/package/tables.py#L5